disruption_py/machine/cmod/physics.py

```diff
@@ -1754,6 +1754,64 @@ def get_sxr_data(params: PhysicsMethodParams):
         sxr = interp1(t_sxr, sxr, params.times)
         return {"sxr": sxr}
 
+    @staticmethod
+    @physics_method(
+        columns=["beta_n"],
+        tokamak=Tokamak.CMOD,
+    )
+    def get_beta_normalized(params: PhysicsMethodParams):
+        """
+        Calculate the normalized beta (beta_n) also known as the Troyon factor.
+
+        beta_n is defined as the total plasma beta (p/(B^2/2mu_0))
+        divided by Ip [MA]/(a*Bt).
+
+        Since the total beta is dominated by the toroidal beta
+        (1/beta_tot = 1/beta_tor + 1/beta_pol), we approximate beta_n using
+        beta_t. This definition is consistent with that of efit_aeqdsk:betan
+        which isn't available for pre-2000 shots. For the same reason we use
+        cpasma and btaxp from AEQDSK instead of ip and btor from magnetics
+        for the plasma current and toroidal field data.
+
+        Parameters
+        ----------
+        params : PhysicsMethodParams
+            The parameters containing the MDSplus connection, shot id and more.
+
+        Returns
+        -------
+        beta_n: the normalized beta given in percentage.
```

Comment:
I'm not sure how we prepared the other docstrings, but we return a dict which holds betan, not betan itself.